plugins/SpectrumAnalyzer/Analyzer.cpp

```diff
@@ -61,15 +62,16 @@ Analyzer::Analyzer(Model *parent, const Plugin::Descriptor::SubPluginFeatures::K
 	// so that it has some reserve space in case data processor is busy.
 	m_inputBuffer(4*4096)
 {
-	m_processorThread = std::thread([=]{m_processor.analyse(m_inputBuffer, m_notifier);});
+	m_processorThread = QThread::create([=]{m_processor.analyse(m_inputBuffer, m_notifier);});
```

Comment:
Qt 5.9 and lower doesn't have `QThread::create`. You may need to create a wrapper class of `QThread` as Qt internally does, see [this](https://github.com/qt/qtbase/blob/c7b3c5a082d9099e7995006f033c3017bfa7f19c/src/corelib/thread/qthread.h#L202-L236) and [this](https://github.com/qt/qtbase/blob/c7b3c5a082d9099e7995006f033c3017bfa7f19c/src/corelib/thread/qthread.cpp#L1076-L1096) for the reference.

Reply:
Ouch, I did not realize the CI system will use a different version, thanks. (I'm on Debian so it didn't even cross my mind that my libraries could be actually newer.. ^^ )

Reply:
Hopefully I got the wrapper working (although it took me longer than I hoped for). To avoid conflicts with existing `QThread::create` implementation on my system I call the wrapper thread  `QThreadCW` ("create workaround"), but otherwise it works in the same way, so it should be easy to remove and replace with the original, once CI upgrades to newer version.

What / who exactly decides which library version is used by the CI system? Is it a Github issue, or is it under our control? It would be nice to have the `::create` function supported, because it seems to be the only supported and reasonably simple way to create a one thread without all the mess that comes with QThread (since `std::thread` does not work because of mingw).